src/main/java/javaslang/collection/Seq.java

```diff
@@ -424,6 +424,20 @@ private boolean checkPrefix(Traversable<T> t, Traversable<T> prefix) {
     Tuple2<? extends Seq<T>, ? extends Seq<T>> splitAt(int n);
 
     /**
+     * Splits a sequence at the first element which satisfies the Predicate, e.g. Tuple(init, element+tail).
+     * @param predicate An predicate
+     * @return A Tuple containing divided sequences
+     */
+    Tuple2<? extends Seq<T>, ? extends Seq<T>> splitAt(Predicate<? super T> predicate);
+
+    /**
+     * Splits a sequence at the first element which satisfies the Predicate, e.g. Tuple(init+element, tail).
+     * @param predicate An predicate
+     * @return A Tuple containing divided sequences
+     */
```

Comment:
Oohh, and please add `{@link #...}` items to the javadoc of the class. (I think we forgot this for other Traversable or Seq methods we added the last days.)

Note: Frequently running `mvn javadoc:javadoc` ensures that doclint is happy.